pkg/sdk/testint/event_tables_integration_test.go

```diff
@@ -0,0 +1,227 @@
+		name := random.String()
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
+
+		err := client.EventTables.Create(ctx, sdk.NewCreateEventTableRequest(id))
+		require.NoError(t, err)
+		t.Cleanup(cleanupTableHandle(t, id))
+
+		set := sdk.NewEventTableSetRequest().WithChangeTracking(sdk.Bool(true))
+		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithSet(set))
+		require.NoError(t, err)
+
+		et, err := client.EventTables.ShowByID(ctx, id)
+		require.NoError(t, err)
+		assert.Equal(t, true, et.ChangeTracking)
+
+		unset := sdk.NewEventTableUnsetRequest().WithChangeTracking(sdk.Bool(true))
+		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithUnset(unset))
+		require.NoError(t, err)
+
+		et, err = client.EventTables.ShowByID(ctx, id)
+		require.NoError(t, err)
+		assert.Equal(t, false, et.ChangeTracking)
+	})
+
+	t.Run("alter event table: set and unset tag", func(t *testing.T) {
+		name := random.String()
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
+
+		err := client.EventTables.Create(ctx, sdk.NewCreateEventTableRequest(id))
+		require.NoError(t, err)
+		t.Cleanup(cleanupTableHandle(t, id))
+
+		set := []sdk.TagAssociation{
+			{
+				Name:  tagTest.ID(),
+				Value: "v1",
+			},
+		}
+		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithSetTags(set))
+		require.NoError(t, err)
+
+		unset := []sdk.ObjectIdentifier{tagTest.ID()}
+		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithUnsetTags(unset))
+		require.NoError(t, err)
+	})
+
+	t.Run("alter event table: rename", func(t *testing.T) {
+		name := random.String()
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
+
+		err := client.EventTables.Create(ctx, sdk.NewCreateEventTableRequest(id))
+		require.NoError(t, err)
+
+		nid := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, random.String())
+		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithRenameTo(&nid))
+		if err != nil {
+			t.Cleanup(cleanupTableHandle(t, id))
+		} else {
+			t.Cleanup(cleanupTableHandle(t, nid))
+		}
+		require.NoError(t, err)
+
+		_, err = client.EventTables.ShowByID(ctx, id)
+		assert.ErrorIs(t, err, collections.ErrObjectNotFound)
+
+		_, err = client.EventTables.ShowByID(ctx, nid)
+		require.NoError(t, err)
+	})
+
+	t.Run("alter event table: clustering action with drop", func(t *testing.T) {
+		name := random.String()
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
+
+		err := client.EventTables.Create(ctx, sdk.NewCreateEventTableRequest(id))
+		require.NoError(t, err)
+		t.Cleanup(cleanupTableHandle(t, id))
+
+		action := sdk.NewEventTableClusteringActionRequest().WithDropClusteringKey(sdk.Bool(true))
+		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithClusteringAction(action))
+		require.NoError(t, err)
+	})
```

Comment:
What about seo alter

Reply:
im not sure i understand what you are asking

Reply:
added search engine optimization set/unset test
```

	t.Run("alter event table: search optimization action", func(t *testing.T) {
		dt := createEventTableHandle(t)
		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, dt.Name)

		action := sdk.NewEventTableSearchOptimizationActionRequest().WithAdd(sdk.NewSearchOptimizationRequest().WithOn([]string{"SUBSTRING(*)"}))
		err := client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithSearchOptimizationAction(action))
		require.NoError(t, err)

		action = sdk.NewEventTableSearchOptimizationActionRequest().WithDrop(sdk.NewSearchOptimizationRequest().WithOn([]string{"SUBSTRING(*)"}))
		err = client.EventTables.Alter(ctx, sdk.NewAlterEventTableRequest(id).WithSearchOptimizationAction(action))
		require.NoError(t, err)
	})
```